apiserver/stateauthenticator/context.go

```diff
@@ -245,10 +246,15 @@ func (ctxt *authContext) externalMacaroonAuth(identClient identchecker.IdentityC
 
 var errMacaroonAuthNotConfigured = errors.New("macaroon authentication is not configured")
 
+const (
+	// TODO make this configurable via model config.
```

Comment:
Will we realistically need this or come back to it? I reckon we should either remove this comment if we're likely never going to do it, otherwise wire it up now.

Reply:
There's been requests to make it configurable at various times. It's not super critical to do at this moment though. I'm +/- 0 on retaining the TODO - we have so many of them already scattered throughout - 1747 at last count :-) What's one more?